deployments/network-constants.ts

```diff
@@ -2,88 +2,88 @@ import constants from './constants';
 
 export default {
   minimumRebalanceInterval: {
-    main: constants.ONE_DAY_IN_SECONDS,
-    kovan: constants.ONE_MINUTE_IN_SECONDS,
+    production: constants.ONE_DAY_IN_SECONDS,
```

Comment:
What does production and staging refer to? This could be simplified to just be one setting per environment instead of mapping production/staging to our environments